elastic/kibana
# [Reporting] Add new `data-render-error` attribute

## Summary

Continuation of https://github.com/elastic/kibana/pull/113544.

We should close out https://github.com/elastic/kibana/issues/103561 after merging this new reporting functionality since these changes should enable improved UX for error reporting from App Services side.

## How to test

1. Start ES + Kibana. Make sure to add `--run-examples` when starting Kibana
2. Go to "Developer examples"
3. Request a new report (one of each kind, legacy and v2)
4. Check the report output

## Follow up work

Once this is merged, we should be able to start working on: https://github.com/elastic/kibana/issues/104395 and close

## Screenshots

### Show render errors in report info panel

<img width="690" alt="Screenshot 2021-10-11 at 13 55 55" src="https://user-images.githubusercontent.com/8155004/136787556-489c67a9-3a60-452b-b21b-182b624e707a.png">


### Checklist
